streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -352,7 +347,7 @@ private void verifyAllWindowedKeys(final List<KafkaStreams> streamsList,
                     final int index = queryMetadata.getActiveHost().port();
                     final KafkaStreams streamsWithKey = pickInstanceByPort ? streamsList.get(index) : streams;
                     final ReadOnlyWindowStore<String, Long> store =
-                        streamsWithKey.store(storeName, QueryableStoreTypes.windowStore());
+                        streamsWithKey.store(storeName, QueryableStoreTypes.windowStore(), true);
```

Comment:
cool. this should now allow standby to be queried

Reply:
Yep!